common/src/attestations/dispute_manager.rs

```diff
@@ -13,7 +13,6 @@ use crate::subgraph_client::{Query, SubgraphClient};
 
 pub fn dispute_manager(
     network_subgraph: &'static SubgraphClient,
-    graph_network_id: u64,
```

Comment:
I know that we've never encountered a case where this is `!= 1`, but maybe it's useful in certain testing scenarios to be able to change that value. So maybe we should keep it a config?

Reply:
I mean, the subgraph itself hardcodes as 1. It's the problem that subgraphs don't have global values so they use 1 as the global key. I took a look at it 👀

Reply:
You can see the code [here](https://github.com/graphprotocol/graph-network-subgraph/blob/036f14b024d5740587a9f3ca19d1b54d1297507d/src/mappings/helpers/helpers.ts#L459)

Reply:
Ok then